fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/utils/NamespaceTools.java

```diff
@@ -75,7 +76,13 @@ public static void validatePath(final Session session, final String path) {
                     try {
                         namespaceRegistry.getURI(prefix);
                     } catch (final NamespaceException e) {
-                        throw new FedoraInvalidNamespaceException("Prefix " + prefix + " has not been registered", e);
+                        if (segment.equals("fcr:backup")) {
```

Comment:
I do not see any way in which the remarks of @awoods is addressing this concern at all. He was talking about your use of `indexOf`.